src/tokenizer/combine.cpp

```diff
@@ -1904,6 +1903,34 @@ void do_symbol_check(Chunk *prev, Chunk *pc, Chunk *next)
          pc->SetType(CT_INCDEC_BEFORE);
       }
    }
+
+   // Issue #3386
+   if (pc->Is(CT_PP_F_PRAGMA))
+   {
+      // such as  _Pragma("P2")
+      LOG_FMT(LFCNR, "%s(%d): GGG orig line is %zu, orig col is %zu, Text() '%s', type is %s\n",
```

Comment:
Is `GGG` in the log message something temporary? If so, we should remove. If not, maybe it would be better to use something a bit more explanatory

Reply:
Yes, thanks you for the catch!